packages/api/src/schema/api-schema.yaml

```diff
@@ -780,6 +802,12 @@ components:
           type: string
           readOnly: true
           example: de7818e7-610a-4057-8f6f-b785dc1e6f88
+        kind:
+          type: string
+          example: stream
+        userId:
+          type: string
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
```

Comment:
> As mentioned above, both SDK and API should have exactly the same responses. Since we have userId on rest api, we would also need for the SDK as well. If we plan to remove this, we should do the same for the rest api as well.